src/main/java/ladder/domain/Name.java

```diff
@@ -0,0 +1,68 @@
+package ladder.domain;
+
+import java.text.MessageFormat;
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+public class Name {
+
+    private static final Pattern NAME_FORMAT = Pattern.compile("[a-zA-Z]+");
+    private static final int NAME_MAX_LENGTH = 5;
+    private static final String NAME_FORMAT_ERROR_MESSAGE = "사람 이름은 영문자만 가능합니다. 현재 입력은 {0} 입니다.";
+    private static final String NAME_LENGTH_ERROR_MESSAGE =
+            "사람 이름은 " + NAME_MAX_LENGTH + "글자까지 가능합니다. 현재 입력은 {0} 입니다.";
+
+    private final String value;
+
+    public Name(final String value) {
+        final String trimValue = value.trim();
+        validate(trimValue);
+        this.value = trimValue;
+    }
+
+    private void validate(final String value) {
+        validateFormat(value);
+        validateLength(value);
+    }
```

Comment:
```suggestion
  private void validate(final String value) {
    if (isNotEnglish(value)) {
      throw new IllegalArgumentException("사람 이름은 영문자만 가능합니다. 현재 입력은" + value + " 입니다.");
    }
    if (hasExceedLength(value)) {
      throw new IllegalArgumentException("사람 이름은 " + NAME_MAX_LENGTH + "글자까지 가능합니다. 현재 입력은+ " + value + "입니다.");
    }
  }
```
argument가 1개일 경우에는 string 연산을 사용해주세요. 
String 재할당 비용보다 비용이 큽니다.

Reply:
`format`에 드는 비용은 생각지도 못하고 있었는데 성능적으로도 생각해볼 수 있었습니다! 감사합니다 👍